Woof/Woof/View/ReusableViews/TextEditorWithPlaceholder.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// A view displaying TextEditor with a prompt text message for a user.
+struct TextEditorWithPlaceholder: View {
+    /// The text to be shown in the TextEditor.
+    @Binding var message: String
+
+    /// The text that is shown in the textEditor when it’s empty.
+    let placeholder: LocalizedStringKey
+
+    var body: some View {
+        ZStack(alignment: .topLeading) {
```

Comment:
What's the purpose of the `ZStack` here?

Reply:
We need it to stacks the placeholderText over TextEditor.  If  I try to removed ZStack I will have the next picture:
<img width="283" alt="Screenshot 2023-06-15 at 6 24 46 PM" src="https://github.com/ios-course/ironfoudation-team-project/assets/120586844/7b494f7d-5b07-4c8f-9b47-85e59e424fab">

Instead of this:
<img width="281" alt="Screenshot 2023-06-15 at 6 27 03 PM" src="https://github.com/ios-course/ironfoudation-team-project/assets/120586844/b2c028b9-cbe2-48fc-a84c-220352de5092">